support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -50,7 +50,7 @@ case class EmailFields(
       DataExtensionName = dataExtensionName,
       SfContactId = userId.left.toOption.map(_.id),
       IdentityUserId = userId.right.toOption.map(_.id),
-      deliveryDate.map(_.toDateTime(new LocalTime(8, 0), DateTimeZone.UTC)),
+      deliveryDate.map(_.toDateTime(LocalTime.now().plusSeconds(90))),//new LocalTime(8, 0), DateTimeZone.UTC)),
```

Comment:
Is this change intentional?

Reply:
no good spot, i thought I unticked that one